### PR TITLE
merge strings for train 42381

### PR DESCRIPTION
**Pull request by bug-mirror**

### CODE REVIEW

File: locale/templates/main.ftl [hunk 1]
```diff
@@ -307,6 +307,10 @@ sub-subscription-error =
 sub-customer-error =
   .title = Problem loading customer
 sub-billing-update-success = Your billing information has been updated successfully
+sub-route-payment-modal-heading = Invalid billing information
+sub-route-payment-modal-copy = There seems to be an error with your Paypal account, we need you to take the necessary steps to resolve this payment issue.
```
Comment (peiying2) [inline locale/templates/main.ftl:311]: Should be `PayPal`, and should be tagged as a brand name.

File: locale/templates/settings.ftl [hunk 2]
```diff
@@ -148,6 +173,22 @@ header-back-to-top-link =
...
+## Modal
+
+modal-close-title = Close
+modal-cancel-button = Cancel
+
+## Modal Verify Session
+
+mvs-verify-your-email = Verify your email
+mvs-enter-verification-code = Enter your verification code
+# This string is used to show a notification to the user for them to enter verification code to verify their email.
```
Comment (peiying2) [inline locale/templates/settings.ftl:185]: add a line break before starting this new section.

File: locale/templates/settings.ftl [hunk 3]
```diff
@@ -24,6 +24,7 @@
 # This is used to refer to a user's account, e.g. "update your Firefox account ..."
 -product-firefox-account = Firefox account
 product-mozilla-vpn = Mozilla VPN
```
Comment (peiying2) [inline locale/templates/settings.ftl:26]: Add the PayPal entry here

File: locale/templates/settings.ftl [hunk 4]
```diff
@@ -24,6 +24,7 @@
...
+product-pocket = Pocket
```
Comment (peiying2) [inline locale/templates/settings.ftl:27]: Add the PayPal entry here.

**Review by peiying2 — CHANGES_REQUESTED** [review]
Need to fix the PayPal entry

File: locale/templates/settings.ftl (hunk 4 above)
Comment (chenba) [reply locale/templates/settings.ftl:27]: `-brand-name-paypal` already exists in `main.ftl`.  `main.ftl` and `settings.ftl` serve two different frontends.  (`main.ftl` is a  confusing name. ☹️ )

**Review by peiying2 — APPROVED** [review]
Thank you for the fix. LGTM now!

**Pull request merged**